Composer/packages/server/src/controllers/publisher.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import pluginLoader from '../services/pluginLoader';
+
+export const PublishController = {
```

Comment:
Probably we should consider make this a REST api managing "publishers" which means:
1. Get /publishers, return a bunches of publisher plugins with ID
2. POST /publishers/:pid/publish, publishing using one publisher
3. POST /publishers/:pid/(rollback/history/status)

Reply:
100% agree, this is part of the work I've done already in another branch!